lib/vantiv_lite/response.rb

```diff
@@ -51,16 +53,31 @@ def each(*args, &blk)
       @to_h.each(*args, &blk)
     end
 
+    def success?
+      @error.nil?
+    end
+
+    def error_message
```

Comment:
Do you ever use the method error_meddage if you already have the instance variable @error ?